src/server/scripts/EasternKingdoms/Karazhan/boss_shade_of_aran.cpp

```diff
@@ -74,32 +75,32 @@ enum Spells
 
 enum Creatures
 {
-    NPC_SHADOW_OF_ARAN  = 18254
+    NPC_SHADOW_OF_ARAN           = 18254
 };
 
 enum SuperSpell
 {
-    SUPER_FLAME = 0,
+    SUPER_FLAME                  = 0,
     SUPER_BLIZZARD,
     SUPER_AE,
 };
 
 enum Groups
 {
-    GROUP_DRINKING      = 0
+    GROUP_DRINKING               = 0
+};
+
+enum Misc
+{
+    ACTION_ATIESH_REACT          = 1,
+    ITEM_ATIESH = 22589
```

Comment:
Firstly, enum spacing.
Second, aren't there four Atiesh staves?
[22589](https://wowgaming.altervista.org/aowow/?item=22589), [22630](https://wowgaming.altervista.org/aowow/?item=22630), [22631](https://wowgaming.altervista.org/aowow/?item=22631), and [22632](https://wowgaming.altervista.org/aowow/?item=22632).